ngraph/core/include/ngraph/op/shuffle_channels.hpp

```diff
@@ -34,7 +34,10 @@ namespace ngraph
             {
             public:
                 static constexpr NodeTypeInfo type_info{"ShuffleChannels", 0};
+                static constexpr NodeTypeInfo type_info_static{"ShuffleChannels", 0};
                 const NodeTypeInfo& get_type_info() const override { return type_info; }
+                static const NodeTypeInfo& get_type_info_static() { return type_info_static; }
```

Comment:
use macros:
1. `NGRAPH_RTTI_DEFINITION(op::v1::ShuffleChannels, "ShuffleChannels", 0);` in implementation 
2. `NGRAPH_RTTI_DECLARATION` in declaration

Reply:
Thanks, done!